src/validators/requiredObject.ts

```diff
@@ -18,9 +18,7 @@ import { ObjectTopLevelError, PrimitiveErrors, Validator } from "../types.ts";
 export function requiredObject(
   message: string,
 ): Validator<object, ObjectTopLevelError<PrimitiveErrors>> {
-  return function requiredValidator(
-    value: object | undefined,
-  ): ObjectTopLevelError<PrimitiveErrors> | undefined {
+  return function requiredValidator(value) {
```

Comment:
Is stripping types really necessary? / the way to go?, isn't `any | undefined | null` what you want?